src/components/AutoSuggest/__tests__/AutoSuggest.test.js

```diff
@@ -390,6 +390,29 @@ describe('src/components/AutoSuggest', () => {
       expect(suggestList).not.toBeInTheDocument();
     });
 
+    test('Enter', async () => {
+      const mockedOnSubmit = jest.fn();
+      const { container } = render(
+        withAppContext(
+          <form onSubmit={mockedOnSubmit}>
+            <AutoSuggest {...props} />
+            <input type="text" name="foo" />
+          </form>
+        )
+      );
+
+      const input = container.querySelector('input[aria-autocomplete=list]');
```

Comment:
It should be possible to use screen to select this input

Reply:
![image](https://user-images.githubusercontent.com/24289432/101366239-00da5500-38a5-11eb-8ac3-6970c38230e7.png)

Reply:
I would gladly do but is not possible :-)